service/pom.xml

```diff
@@ -46,6 +47,12 @@
                 <artifactId>postgresql</artifactId>
                 <version>${postgrees.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
+                <version>${assertj.version}</version>
+                <scope>test</scope>
```

Comment:
move scope to <dependencies> block from <dependencyManagement>

Reply:
Done